### ISSUE TITLE
What is WordPress?

### ISSUE THREAD

**Issue by desrosj** [WordPress/Marketing-Team#205]
I was recently sending out some emails to potential local meetup venues and I was looking for an official 1-2 sentence paragraph explaining the software that I could use as a starting point in my emails to explain the meetup group.

I noticed that the wordpress.org home page (and the entire site that I could tell) does not actually say what WordPress is. There's a lot of examples of what it can do or how it can be used, but no boiled down factual summary of "what is WordPress?". If I were someone who had no idea what WordPress was and landed on the home page, I'm not sure I'd be able to figure that out. Even on the about page, the Our Story section is a bit vague on these details.

**Comment by josephahaden** [comment]
That's a great call out, Jonathan. At minimum, we should probably add:

"WordPress is a free and open source content management system (CMS) [...]." 

Then I would imagine some contextual color as well. Something that suggests its flexibility of size/purpose, but then brings it back around to plain language if folks just want to know "how to get online".

**Comment by sereedmedia** [comment]
This is a very existential and important issue, and I agree that it should be defined! I personally explain WordPress as a platform for building and managing websites. Of course the free and open source part @josephahaden brought up is crucial, so I would expand that to:

""WordPress is a free and open source platform for building and managing websites."

We could then have a "learn more about WordPress" link that leads to https://wordpress.org/about/

**Comment by pbiron** [comment]
Especially in the context Jonathan is talking about, using wording like Sé suggests is better than "CMS" (because those who don't know what WP is probably won't know know what a "content management system (CMS)" is :confused:

**Comment by jennimckinnon** [comment]
“WordPress is a free, open source platform for building and managing websites and blogs.”
[Discover More] (button)

**Comment by rmartinezduque** [comment]
This is certainly a great call. Thanks for sharing! If the goal is to find and "convince" venues to help support meetup groups by letting them host the events, I would be inclined to add some more context that helps build credibility/trust. Perhaps something along these lines:

> WordPress is a free and open source content management system (CMS) for building and managing websites. Known for its flexibility and simplicity for getting online, WordPress is the platform of choice by millions of websites and users worldwide—from bloggers and small businesses to large corporations and media outlets.

Additionally, to introduce the community aspect, something like this could be added:

> Supporting its [mission of democratizing publishing](https://wordpress.org/about/), there's a large community of people collaborating on and contributing to the project.

**Comment by sereedmedia** [comment]
@rmartinezduque I really like your example, but will comment on that separately. 

I believe the goal of the ticket is to address the lack of the clear/succinct definition of WordPress on the homepage of wordpress.org.

Introducing venue owners to WordPress was what led the OP to look for that definition on wordpress.org. (@desrosj please correct me if I'm wrong on that!)

**Comment by sereedmedia** [comment]
In terms of the top-of-line definition, the audience for the first touch of the homepage certainly includes new users, new developers/agencies, and new clients of developers/agencies/community. 

To that end, I think we can assume a good baseline for this would be a layperson's familiarity with industry terms. (And anyway since WP language is intended to be at a widely readable level, this is a safe assumption.)

"Free and open source" is already a relatively unknown, not easily defined, potentially confusing phrase for a lot of people (including developers!).

That said, I think we **have** to say "free and open source". It's the bedrock of all of this _gestures around_. It is the core of the WordPress ethos and it really needs to have top billing. 

So then we have "content management system (CMS)" - another relatively unknown, not easily defined, and potentially off-putting phrase (in English).

I think having "free and open source" and "content management system" next to each other in this context presents an immediate obstacle to that baseline understanding. 

However, because of the ubiquity of streaming platforms and gaming platforms, the word "platform" is very accessible. People know what a "platform" is: Please see attached image of Google trend displaying "platform" vs "content management system".

![image](https://user-images.githubusercontent.com/28426780/226091691-845154f5-dc61-4ebb-ad40-0b649c30e563.jpeg)




_________________
_________________

In terms of internationalization, ChatGPT had this to say:

The word "platform" in the context of "website platforms" is a common term used in many languages, including Spanish, French, German, and Portuguese. However, in some languages, there may be more specific or nuanced terms that are more appropriate for describing a website platform.

For example, in Japanese, the term "web platform" is often translated as ウェブプラットフォーム (webyu purattofoomu), which is a direct transliteration of the English term. However, there are other terms in Japanese that could be used to describe different aspects of a website platform, such as ウェブサービス (webu saabisu) for web services, or ウェブアプリケーション (webu apurikeeshon) for web applications.

**Comment by rmartinezduque** [comment]
> I believe the goal of the ticket is to address the lack of the clear/succinct definition of WordPress on the homepage of wordpress.org.

Yes, thanks @sereedmedia. My mind went a bit further, and I simply thought of a short boilerplate that could be used/adapted on the website (not necessarily on the homepage) but also come in handy for the particular use case Jonathan commented (people looking for that "official 1-2 sentence paragraph" on the site to explain/pitch what WordPress is to others). Either way, it's just a suggestion. :)

**Comment by sereedmedia** [comment]
@rmartinezduque totally! i agree we need/should have that boilerplate content! and i think the wording and clarity in your suggestion is great! 

incidentally, that further boilerplate issue reminds me of a very old ticket I found that deals that discusses/drafts boilerplate Five for the Future explanations. i'll find that ticket and link it here for reference.

**Comment by courtneyr-dev** [comment]
Platform does sound succinct to me. Can we check in with Polyglots folks to consider translation implication for several of our main languages?

**Comment by nomad-skateboarding-dev** [comment]
A personal reflection: the only time I've ever used CMS, or said that to others, was appending it to WordPress. I've never had a client or someone I was speaking to mention CMS or content management systems.

"Platform" sounds good to my ear. But any other word, or dropping the CMS, I'm in favor of.

**Comment by josephahaden** [comment]
Refining the example that @rmartinezduque shared and pulling in @sereedmedia's research:

> WordPress is a free and open source software for building and managing websites. Known for its flexibility and simplicity in getting online, WordPress is the platform of choice for millions of websites and users worldwide—from bloggers and small businesses to large corporations and media outlets. Find out more about the web's favorite CMS here. [Discover More] (button)

I put "software" (a very well-known descriptor) by free and open source, then introduced "platform", and finally introduced "CMS" with a button.

Does that meet people where they are and point them where they need to go? Or does it look like we can't decide on a term? I think "Find out more about the web's favorite CMS here." is negotiable, but maybe not the button.

**Comment by jaapwiering** [comment]
I agree that CMS needs a place in the description, but I don't like "web's favorite". To my opinion you don't have to  emphasive on your success.

I like the idea of a button, but [Discover more] is not descriptive enough from an accessability perspective. It should be something like [Discover more features of WordPress].

**Comment by sereedmedia** [comment]
I think that's a great version, @josephahaden, especially in terms of the sfotware/platform phrasing. 

I definitely think WordPress is known for its flexibility. However, I wonder if it is actually the case that WordPress is known for its simplicity in getting online. In my experience, it is known as being somewhat confusing to the average user. So perhaps we could focus on some of WordPress's other strengths for that segment, such as "Known for its flexibility and ease of use" or "known for being flexible and user-friendly".

Also agree with @jaapwiering that "web's favorite" sounds like marketing hype. How about something like: **Find out more about the CMS that powers 43% of the web.** (There truly is not a more relevant place for that statistic but on the dot-org homepage.) 

Building those comments in (plus a tad more on the types of users):

> WordPress is a free and open source software for building and managing websites. Known for being flexible and user-friendly, WordPress is the platform of choice for millions of websites and users worldwide—from bloggers and small businesses to large corporations, media outlets, and even governments. _Find out more about the CMS that powers 43% of the web._ [text link to about page]

**Comment by jasmussen** [comment]
Josepha's version looks good to me. 

As far as implementation into designs, we can be flexible, but generally what works best here is making it as short and portable as possible. The fewer words, the larger we can show the text, and the more likely it is to be read in its entirety.

**Comment by tomxygen** [comment]
Comparing .com and .org home pages, it's completely clear what wordpress is on .com, the word's most popular website builder, but it makes little sense on .org. Publish your passion? Is it a streaming platform like youtube? Or a forum where I can find a community of people with my passions?
<img width="1549" alt="Screenshot 2023-05-01 at 12 14 07 PM" src="https://user-images.githubusercontent.com/47855768/235439537-aeffd5f1-3842-4a67-8159-37e2aa4bc129.png">

<img width="1549" alt="Screenshot 2023-05-01 at 12 14 00 PM" src="https://user-images.githubusercontent.com/47855768/235439516-2bb3a546-910e-421a-950a-9d6b3dcd1a87.png">

**Comment by jpantani** [comment]
@josephahaden's boilerplate description of WordPress checks a lot of boxes and puts this discussion in a good place. 
I think, however, we run into some issues with the semantics of a few words: 

1. popular (measurable and stable) vs 
2. favorite (subjective yet if WordPress can't say it about itself who will?) vs 
3. a stat/43% (measurable and movable, yet volatile and source dependent) 

Putting a number in the boilerplate carries a bunch of risks. The number is subject to change and makes it difficult to update in all the places where this will be used. People fixate on numbers and it becomes a constant source of worry(I know there are other discussions about this number and depending on the source you check WP's popularity differs.) Plus, if the number were to dip, that is not a good look for the brand. 43% is only good if WP continues to increase its share.

Favorite as mentioned above adds subjectivity and where there is subjectivity skepticism can be found. Best to avoid.

Vague and superlative don't necessarily need to sound like hype though. Something positive-neutral sounding like "Discover what the web's most popular CMS can do for you" could very well accomplish these three things: 

1. reaffirming WP's pole position 
2. continuing trust because it implicitly says it's trusted by millions and 
3. popular means it's counted by people who choose it and directly refers back to the community, another core pillar of WordPress.org and its ecosystem.

**Comment by jonoalderson** [comment]
Can we combine @sereedmedia's tweaks with @jpantani's tweaks? 

That gives us something like this (with some additional de-fluffing and consolidation):

> WordPress is free, open-source software for building and managing websites. It's flexible, user-friendly, and powers millions of websites worldwide - from bloggers and small businesses to large corporations, media outlets, and even governments. [Learn more](https://wordpress.org/about/) about the CMS & platform that powers over 40% of the web, or [download the software](https://wordpress.org/download/) and [get started](https://wordpress.org/documentation/article/get-started-with-wordpress/).

(though I reckon that 'get started' article needs to be a bit more... humanized)

**Comment by jpantani** [comment]
This is getting closer and closer. Separating Software, Platform, and CMS into three sentences has a distinct advantage by refining what the software does and bringing people along in a succinct way. 

Building off @jonoalderson's suggestion. 
WordPress is free, open-source software for building and managing websites. Its flexible and user-friendly platform sustains millions of websites worldwide - from bloggers and small businesses to enterprise sites and governments. Learn more about the CMS that powers most of the web [or more of the web than any other] and how to get started today. 

In the end, it's a boilerplate so it won't be the most natural-sounding summary out there but covers a vast majority of what we'd like it to.

**Comment by ngwebpt** [comment]
@desrosj We're at #WordCampLisboa2023 and decided to leave our insights about this issue. 

First thoughts: Look at it as from the client perspective, of someone who has no idea what WordPress is or what the possibilities of this tool (or platform) has. 

We must differentiate between the websites WordPress.com and WordPress.org. 
In our perspective, wordpress.com is for people who plan to have an easier solution for building and launching website in a fast way, and for this, wordpress.com has the plans and pricing so people can choose and start immediately. 

On the other hand, wordpress.org is free, but is mainly for technical users who rather install on their own hosting. 

If we look at the speed of information consumption nowadays, is fast, and mainly video and /or images with short and direct messages to clearly pass key information. 

With that said, our suggestions are: 

1. Renew completely the homepage for wordpress.org. 
2. Insert a video or slideshow as hero section, with the key information of what wordpress.org can offer, like: 
         - more than 43% of the World websites use WordPress
          - For Your Corporate Website (CTA - Know more - link to examples)
          - For Your Learning System (online courses) - (CTA - See How - link to examples)
          - For Your E-commerce Solution (CTA - Sell Your Products with WordPress - link to examples)
          - CTA - Discover How to Start

The reason behind having a broader approach to the possibilities ( including e-commerce and LMS) and showing it from the beginning, is purely because it's a reality of the WordPress implemented solutions nowadays. 

This video / slides should have nice and clear images that will help people understand the key information points. 

Moreover, we suggest having a CTA to "What's new in WordPress" which will resume the main new features of the latest WordPress version. 

We didn't mention anywhere the expressions, "CMS - Content Management System" and "Open Source" because at this stage those in our opinion are not needed.

Natacha Gomes and João Tiago Ogando (jtogando@gmail.com) 

João Tiago Ogando is a WordPress user, such as myself, with extensive knowledge of WP, and we are both at #WordCampLisboa2023.

**Comment by Lennair19** [comment]
This is a great subject. I would like to add that it could be beneficial to make the differentiation of both WordPress and I believe it would be helpful to write the name like this: WordPress.org and WordPress.com right on the homepages. Many people get confused with both of them and do not understand the differences. I believe adding .org and .com at the end can help to understand better what WordPress is and how big it is.

**Comment by flexseth** [comment]
**About WordPress**
WordPress is a semantic publishing platform that enables website operators and users to create content for display across the web and a variety of mobile devices.

**WordPress.com**
WP.com is a great option for those just starting out, or those who do not want to worry about having to do anything to "host" WordPress. This relieves the duties of having to update plugins and themes, but comes with the limitation that there are a limited, but utility, set of plugins and themes available. There are free options to get started out, or if you're looking for a more robust set of features, paid plans are available. 

**WordPress.org**
The central place to find contributed themes and plugins, and expansive documentation on how to modify and develop on top of the WordPress platform. When hosting with a provider other than WordPress.com, a website administrator will have full control of their website and the ability to add a vast array of plugins and themes that have been developed and vetted. 

WordPress.org provides tools in the form of plugins, themes, and documentation that allows advanced users to create rich user interfaces, take advantage of web services, and even use the platform solely to relay data to other websites.

**Comment by jtogando** [comment]
> @desrosj We're at #WordCampLisboa2023 and decided to leave our insights about this issue.
> 
> First thoughts: Look at it as from the client perspective, of someone who has no idea what WordPress is or what the possibilities of this tool (or platform) has.
> 
> We must differentiate between the websites WordPress.com and WordPress.org. In our perspective, wordpress.com is for people who plan to have an easier solution for building and launching website in a fast way, and for this, wordpress.com has the plans and pricing so people can choose and start immediately.
> 
> On the other hand, wordpress.org is free, but is mainly for technical users who rather install on their own hosting.
> 
> If we look at the speed of information consumption nowadays, is fast, and mainly video and /or images with short and direct messages to clearly pass key information.
> 
> With that said, our suggestions are:
> 
> 1. Renew completely the homepage for wordpress.org.
> 2. Insert a video or slideshow as hero section, with the key information of what wordpress.org can offer, like:
>    - more than 43% of the World websites use WordPress
>    - For Your Corporate Website (CTA - Know more - link to examples)
>    - For Your Learning System (online courses) - (CTA - See How - link to examples)
>    - For Your E-commerce Solution (CTA - Sell Your Products with WordPress - link to examples)
>    - CTA - Discover How to Start
> 
> The reason behind having a broader approach to the possibilities ( including e-commerce and LMS) and showing it from the beginning, is purely because it's a reality of the WordPress implemented solutions nowadays.
> 
> This video / slides should have nice and clear images that will help people understand the key information points.
> 
> Moreover, we suggest having a CTA to "What's new in WordPress" which will resume the main new features of the latest WordPress version.
> 
> We didn't mention anywhere the expressions, "CMS - Content Management System" and "Open Source" because at this stage those in our opinion are not needed.
> 
> Natacha Gomes and João Tiago Ogando ([jtogando@gmail.com](mailto:jtogando@gmail.com))
> 
> João Tiago Ogando is a WordPress user, such as myself, with extensive knowledge of WP, and we are both at #WordCampLisboa2023.

Well, after so Much time I still see the same confusing websites. No sign of changes!
Maybe is just lack of "Political" will...?

Hope you make progresses!

**Comment by tomxygen** [comment]
> WP.com is a great option for those just starting out
I don't necessarily agree with that, or at least, WP.com is not limited to that kind of user.
I have a Woo store that is doing around $1.5M/year and we have high bursts of traffic due to much of it coming from Influencers.
It was previously hosted on AWS but I experienced several slowdowns and crashes, and then I switched to WP.com. I'm paying a 10th of what I was paying on AWS and I haven't experienced a single slowdown ever since, thanks to their scalable architecture that gives virtually unlimited scalability.
We haven't left WP.com ever since, tho we switched plan to Woo Express (still on WP.com) when it was released a couple of months ago.

**Comment by jeffpaul** [comment]
I appreciate all the iterations getting to a well-defined description.  Alternatively I'll point out that if someone wanted a much more concise option (which could reference back to a canonical, more complete description) that they could use what's in [WordPress' `package.json` file](https://github.com/WordPress/wordpress-develop/blob/68fd1e3b043e14e2c5c8e6246cfee172c3eeceb1/package.json#L4):
> WordPress is open source software you can use to create a beautiful website, blog, or app.

**Comment by aaronjorbin** [comment]
I think we should close this with @jeffpaul's suggestion of using what has been used since [2018](https://core.trac.wordpress.org/changeset/42961) in package.json.

**Comment by monchompa** [comment]
Just thought I'd give my 2 cents on the messaging of WordPress.org. I agree with the callouts from Jonathan. When I look at the page I don't see the real story of what WordPress is. 

Think of these questions:
What is it?
Why should I try it?
How much does it cost?

If the page doesn't tell that story, then we may need to think about a messaging roadmap and focus on the basics of who is the target audience, what problems WordPress solves, and how economical is it for everyday users. 

A document outlining the following would be a good place to start:

1. Define target audience
2. Define customer challenges
3. Examples of how WordPress Solves those challenges
4. Competitive threats
5. 3-5 Reasons to buy
